java/google/registry/whois/Whois.java

```diff
@@ -26,18 +26,22 @@
 
   private final Clock clock;
   private final String disclaimer;
+  private final WhoisCommandFactory commandFactory;
 
   @Inject
-  public Whois(Clock clock, @Config("whoisDisclaimer") String disclaimer) {
+  public Whois(Clock clock,
```

Comment:
The line-wrapping/indentation is wrong.  Use google-java-format on this block to fix.